packages/console/src/utilities/translation.ts

```diff
@@ -0,0 +1,9 @@
+import i18next from 'i18next';
+
+export const translateUnnamed = (resource: Record<string, string>) => {
+  const { languages } = i18next;
+  const matchedLanguage = languages.find((language) => resource[language]);
```

Comment:
matchedTranslation better? as it if returning the actually content not a language.

Reply:
updated.